fixtures/vitest-pool-workers-remote-bindings/run-tests.mjs

```diff
@@ -58,12 +60,19 @@ const deployOut = execSync(
 		stdio: "pipe",
 		cwd: "./.tmp",
 		env,
+		encoding: "utf8",
 	}
 );
+
 if (!new RegExp(`Deployed\\s+${remoteWorkerName}\\b`).test(`${deployOut}`)) {
 	throw new Error(`Failed to deploy ${remoteWorkerName}`);
 }
 
+const deployedUrl = deployOut.match(
+	/(?<url>https:\/\/tmp-e2e-.+?\..+?\.workers\.dev)/
```

Comment:
Like my other comment I wonder if it would be more generic and robust to use `remoteWorkerName` here 🤔